lib/execution_engine2/utils/SDKMethodRunner.py

```diff
@@ -706,3 +706,52 @@ def start_job(self, job_id, ctx, skip_estimation=True):
 
         with self.get_mongo_util().mongo_engine_connection():
             job.save()
+
+    def check_job(self, job_id, ctx):
+        """
+        check_job: check and return job status for a given job_id
+
+        Parameters:
+        job_id: id of job
+        """
+        logging.info("Start fetching status for job: {}".format(job_id))
+
+        if not job_id:
+            raise ValueError("Please provide valid job_id")
```

Comment:
I think we should probably filter the output of this at some point.. I don't think we want to return the entire 'output' object.. Or do we ? It might be super slow.

Reply:
👍 added project field. I don't know if we want to by default filter out output. maybe we do. Think for now, it's fine returning whole Job.

Reply:
Maybe we should combine check_job into check_jobs into one function, and expect an array to always be passed in
Also instead of using get_mongo_util.get_job, we should write a new one called get_jobs() that expects an array, and does a WHERE * in [job_id1,jobid2] instead of WHERE * = job_id
Also instead of deleting the keys, you can filter the query https://stackoverflow.com/a/33271295

Reply:
👍 sure.